From: Youhei SASAKI <uwabami@gfd-dennou.org>
Date: Sun, 2 Jun 2013 02:40:18 +0900
Subject: Remove RubyGems Depends

Signed-off-by: Youhei SASAKI <uwabami@gfd-dennou.org>
---
 test/test_helper.rb |    2 --
 1 file changed, 2 deletions(-)

diff --git a/test/test_helper.rb b/test/test_helper.rb
index d98e02e..86ebe4b 100644
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,8 +1,6 @@
 # coding: UTF-8
 Encoding.default_internal = 'UTF-8' if defined? Encoding
 
-gem 'test-unit', '>= 2' # necessary when not using bundle exec
-
 require 'test/unit'
 require 'nokogiri'
 
