From: Youhei SASAKI <uwabami@gfd-dennou.org>
Date: Thu, 27 Feb 2014 21:22:41 +0900
Subject: Fix markdown_test

Signed-off-by: Youhei SASAKI <uwabami@gfd-dennou.org>
---
 test/markdown_test.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: ruby-redcarpet/test/markdown_test.rb
===================================================================
--- ruby-redcarpet.orig/test/markdown_test.rb
+++ ruby-redcarpet/test/markdown_test.rb
@@ -306,6 +306,7 @@ text
 
   def test_references_with_tabs_after_colon
     markdown = @markdown.render("[Link][id]\n[id]:\t\t\thttp://google.es")
-    html_equal "<p><a href=\"http://google.es\">Link</a></p>\n", markdown
+    html_equal '<p><a href="http://google.es">Link</a></p>
+', markdown
   end
 end
